internal/core/data/operators/value_descriptor/get_test.go

```diff
@@ -95,7 +95,7 @@ func TestGetValueDescriptorsByNames(t *testing.T) {
 			config.ServiceInfo{MaxResultCount: 1},
 			nil,
 			true,
-			&errors2.ErrLimitExceeded{},
```

Comment:
While we're in the neighborhood, should we update this from `errors2`?

Reply:
Good point, I think I will follow what you have done recently and alias the `builtin/errors` with `goErrors` and remove the alias from our errors.